test/sharness/t0020-service-startup-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# MIT LICENSED
+#
+test_description="Test service startup and init functionality"
+
+. lib/test-lib.sh
+
+test_expect_success "ipfs is installed on this machine" '
+   ipfs help >ipfs_help.txt &&
+   egrep -i "^Usage" ipfs_help.txt >/dev/null  
+'
+
+test_expect_success "launch ipfs daemon" '
+   mkdir -p ../.test_ipfs &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs init'"'"' &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs daemon & echo $! >../dPID.txt'"'"' &&
+   sleep 2 
+'
+
+test_expect_success "test config folder exists" '
```

Comment:
Test your prerequisite for ipfs is true. That should imply that it is running with the test folder.